src/locales/pt.js

```diff
@@ -0,0 +1,706 @@
+    'menu.pageModels': 'Modelos de Página',
+    'menu.pageModelDetails': 'Detalhes do Modelo de Página',
+    'menu.uxPattern.widget': 'Widgets',
+    'menu.uxPattern.fragment': 'Fragmentos',
+    'menu.pageSettings': 'Detalhes de Página',
+    'menu.pageConfig': 'Configuração de Página',
+    'menu.dataType': 'Tipos de Dado',
+    'menu.userManagement': 'Gestão de Usuários',
+    'menu.users': 'Usuário',
+    'menu.users.authority': 'Autorizações',
+    'menu.users.restrictions': 'Restrições de Usuário',
+    'menu.cms': 'CMS',
+    'menu.cms.contentList': 'Lista de Conteúdo',
+    'menu.cms.contentTypes': 'Tipos de Conteúdo',
+    'menu.cms.contentModels': 'Modelos de Conteúdo',
+    'menu.cms.contentSettings': 'Configurações de Conteúdo',
+    'menu.details': 'Detalhes',
+    'menu.groups': 'Grupos',
+    'menu.database': 'Banco de Dados',
+    'menu.widget': 'Widget',
+    'menu.labelsAndLanguages': 'Labels e Idiomas',
+    'menu.uxPattern.addLabels': 'Rótulos de sistema',
+    'menu.categories': 'Categorias',
+    'menu.roles': 'Papéis',
+    'menu.reloadConfiguration': 'Recarregar Configuração',
+    'menu.reloadConfirm': 'Recarregar a Configuração',
+    'menu.profileTypes': 'Tipos de Perfil',
+    'menu.profile': 'Perfil',
+    'dashboard.newUser': 'Novo Usuário',
+    'dashboard.newWidget': 'Novo Widget',
+    'dashboard.newPage': 'Adicionar Página',
+    'dashboard.apis': 'APIs',
+    'dashboard.numberWidgets': 'Número de Widgets',
+    'dashboard.languages': 'Idiomas',
+    'dashboard.activeLanguages': 'Idiomas Ativos',
+    'pageTree.pageTree': 'Árvore de paǵinas',
+    'pageTree.expand': 'Expandir',
+    'pageTree.collapse': 'Retrair',
+    'pageTree.status': 'Status',
+    'pageTree.displayedInMenu': 'Mostrado no menu',
+    'pageTree.actions': 'Ações',
+    'pageTree.action.clear': 'Limpar resultados',
+    'pageTree.searchForm.title': 'Buscar por uma página existente',
+    'pageTree.searchForm.code': 'Código de Página',
+    'pageTreePage.help': 'A seção ÁRVORES DE PÁGINAS permite você gerenciar uma página existente, mostrar ou esconder todas as páginas existentes na árvore ou mostrar todas/esconder todas que pertencem ao nó principal. Você pode também adicionar ou mover uma página, dentre outras ações: preview mobile ou desktop, configurar, editar, ver detalhes, clonar umá página ou deletar. Além disso você pode visualizar o estado (rascunho ou online) se a página estiver na lista de menu.',
+    'pageDetails.title': 'Detalhes de Página',
+    'pageDetails.help': 'A seção ÁRVORES DE PÁGINAS permite você gerenciar uma página existente, mostrar ou esconder todas as páginas existentes na árvore ou mostrar todas/esconder todas que pertencem ao nó principal. Você pode também adicionar ou mover uma página, dentre outras ações: preview mobile ou desktop, configurar, editar, ver detalhes, clonar umá página ou deletar. Além disso você pode visualizar o estado (rascunho ou online) se a página estiver na lista de menu.',
```

Comment:
I think here we should have `Detalhes da Pagina` instead of `ÁRVORES DE PÁGINAS`